seatunnel-core/seatunnel-spark-starter/src/main/java/org/apache/seatunnel/core/starter/spark/execution/SourceExecuteProcessor.java

```diff
@@ -57,6 +57,7 @@ public List<Dataset<Row>> execute(List<Dataset<Row>> upstreamDataStreams) {
             Dataset<Row> dataset = sparkEnvironment.getSparkSession()
                 .read()
                 .format(SeaTunnelSource.class.getSimpleName())
+                .option(Constants.PARALLELISM, sparkEnvironment.getSparkConf().get(Constants.PARALLELISM))
```

Comment:
Can parallelism be set for a single plugin? If the plugin don‘t set the parallelism, get the global parallelism.

Reply:
Resolved.